examples/requirements.txt

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
https://github.com/inveniosoftware/invenio-files-rest/blob/master/setup.py#L59-L67 should probably include versioning - but just keep the `requirements.txt` for now